src/MultiRegion/multi_region_cubed_sphere_grid.jl

```diff
@@ -210,41 +210,100 @@ function ConformalCubedSphereGrid(arch::AbstractArchitecture=CPU(), FT=Float64;
                                                                                            region_grids,
                                                                                            devices)
 
-    fields₁ = (:Δxᶜᶜᵃ,  :Δxᶠᶜᵃ,  :Δxᶜᶠᵃ,  :λᶜᶜᵃ,   :λᶠᶜᵃ,   :λᶜᶠᵃ,   :Azᶜᶜᵃ,  :Azᶠᶜᵃ)
-    LXs₁    = (:Center, :Face,   :Center, :Center, :Face,   :Center, :Center, :Face)
-    LYs₁    = (:Center, :Center, :Face,   :Center, :Center, :Face,   :Center, :Center)
+    # the following fills the coordinate and metric halos using halo-filling algorithms
+    ccacoords = (:λᶜᶜᵃ, :φᶜᶜᵃ)
+    fcacoords = (:λᶠᶜᵃ, :φᶠᶜᵃ)
+    cfacoords = (:λᶜᶠᵃ, :φᶜᶠᵃ)
+    ffacoords = (:λᶠᶠᵃ, :φᶠᶠᵃ)
```

Comment:
@siddharthabishnu you seem to undo here the fixes I made in #3256

Reply:
After you merged the PR associated with #3256, I merged the main branch into this one, and accepted the 'incoming changes'. I just wanted to incorporate those changes, not undo them. I am not sure how that happened.

Reply:
Okie then! No worries